test/compress/loops.js

```diff
@@ -213,6 +213,30 @@ evaluate: {
             a();
         for(;;)
             c();
-        d();
+        // rule disabled due to issue_1532
```

Comment:
// rule disabled due to issue #1532

would be more hyperlink friendly in github browsing.